Example/TABTestKit/ViewController.swift

```diff
@@ -9,15 +9,59 @@
 //
 
 import UIKit
+import LocalAuthentication
 
 class ViewController: UIViewController {
-  @IBOutlet var helloWorldLabel: UILabel!
-  @IBOutlet var testTextField: UITextField!
+  @IBOutlet private var helloWorldLabel: UILabel!
+  @IBOutlet private var testTextField: UITextField!
+  @IBOutlet private var biometricsStateLabel: UILabel!
+  @IBOutlet private var authenticateButton: UIButton!
   
-    override func viewDidLoad() {
-      helloWorldLabel.text = "Hello world!"
-      testTextField.accessibilityIdentifier = "Test Text Field"
-      helloWorldLabel.accessibilityIdentifier = "Hello World Label"
+  @IBAction private func authenticateButtonTapped(_ sender: UIButton) {
+    let context = LAContext()
+    context.evaluatePolicy(.deviceOwnerAuthenticationWithBiometrics, localizedReason: "Authenticate now!", reply: { _, _ in })
+  }
+  
+  override func viewDidLoad() {
```

Comment:
Missing a call to `super.viewDidLoad()`. Realise that was missing before so that's my bad

Reply:
Fixed!